### PR TITLE
:lipstick:News #7

### PR DESCRIPTION
**Pull request by SKaranjaN**
# Description

Created the new blog page which is  also mobile responsive.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers #7 (https://github.com/italanta/elewa-website-html/issues/7).

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Screenshot
image
image

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -0,0 +1,823 @@
...
+  padding-top: 25px;
+  padding-left: 70px;
+}
+
+
+.nav-options {
+  text-align: center;
+  margin-top: -50px;
+  margin-left: -10px;
+}
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: use margin shorthand

File: src/css/style.css [hunk 2]
```diff
@@ -0,0 +1,823 @@
...
+  color: #000;
+}
+
+
+/* solutions drop-down items */
+
+.dropdown {
+  position: relative;
+  display: inline-block;
+  transition: 2s;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: this doesent take long?

File: src/css/style.css [hunk 3]
```diff
@@ -0,0 +1,823 @@
...
+
+.group-menu {
+  display: flex;
+  justify-content: space-between;
+}
+
+.menu {
+  border-radius: 10px;
+  margin-right: 8px;
+  margin-left: 8px;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: shorthand

File: src/css/style.css [hunk 4]
```diff
@@ -0,0 +1,823 @@
...
+  height: 150px;
+  width: auto;
+  overflow: hidden;
+  padding-top: 15px;
+}
+.menu-text{
+  text-align: left;
+  margin-left: -10px;
+  margin-right: 10px;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: here also

File: src/css/style.css [hunk 5]
```diff
@@ -0,0 +1,823 @@
...
+  margin-left: 20px;
+}
+
+/* hover effect on the dropdown containers */
+
+.menu:hover {
+  background-color: #f8f3f3;
+  border-radius: 10px;
+  margin-right: 8px;
+  margin-left: 8px;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: here

File: src/css/style.css [hunk 6]
```diff
@@ -0,0 +1,823 @@
...
+  margin-left:8px;
+}
+
+.Consultuncy-img-2 {
+  background-image: linear-gradient(90deg, #fff 0%, #c0c2fa 55%, #a3a6ec 80%);
+  height: 40px;
+  width: 40px;
+  border-radius: 50%;
+  margin-left: 33px;
+  margin-top: -50px;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: here

File: src/css/style.css [hunk 7]
```diff
@@ -0,0 +1,823 @@
...
+  background-image: linear-gradient(
+  180deg,
+  rgba(241, 209, 149, 1) 0%,
+  rgba(255, 238, 207, 0.6) 50%,
+  rgba(255, 255, 255, 0) 80%);
+  height: 30px;
+  width: 30px;
+  border-radius: 50%;
+  margin-left: 10px;
+  margin-top: 40px
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: here

**Review by JAPHETHNYARANGA — CHANGES_REQUESTED** [review]
Very good attempt. Change the pr emoji to a lipstick. Then push only the required files, use shorthand for margin.

File: src/css/style.css (hunk 3 above)
Comment (SKaranjaN) [reply src/css/style.css]: I have made the requested changes.

**Comment by JAPHETHNYARANGA** [comment]
This is good work, the design is nice.Good work. Just closing the pr since the issue is closed

**Pull request closed without merging**